great_expectations/expectations/core/expect_table_columns_to_match_set.py

```diff
@@ -34,12 +34,15 @@ class ExpectTableColumnsToMatchSet(BatchExpectation):
     expect_table_columns_to_match_set is a \
     [Batch Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations).
 
+    BatchExpectations are one of the most common types of Expectation.
+    They are evaluated for an entire Batch, and answer a semantic question about the Batch itself.
+
     Args:
         column_set (list of str): \
             The column names, in any order.
         exact_match (boolean): \
             If True, the list of columns must exactly match the observed columns. \
-            If False, observed columns must include column_set but additional columns will pass.
+            If False, observed columns must include column_set but additional columns will pass. Default True.
```

Comment:
```suggestion
           When the value is False, observed columns must include column_set, but additional columns pass. The default is True.
```
Try to provide complete sentences with articles as they are easier to parse quickly.